apps/mobile/src/screens/welcome/welcome.tsx

```diff
@@ -13,8 +14,18 @@ import {
 	type QuestionsStackNavigationParameterList,
 } from "~/libs/types/types";
 
+import {
+	LETTERS_PRINTED_PER_STEP,
+	NEXT_LETTERS_DELAY,
+	NEXT_PARAGRAPH_DELAY,
+} from "./libs/constants/constants";
 import { styles } from "./styles";
 
+const paragraphs = [
+	"Thank you for sharing your insights! We’re currently processing your responses to create a personalized path just for you. This is where the magic begins—we’re using your input to tailor the experience, offering you the guidance and motivation you need to achieve a balanced, fulfilling life.",
+	"Hang tight while we set things up! In just a moment, you’ll dive into the areas that matter most to you, and together, we’ll start making progress toward your goals. Your journey to a better life starts now!",
+];
+
```

Comment:
Don't we need to add a blanc line between paragraphs?
```suggestion
const paragraphs = [
	"Thank you for sharing your insights! We’re currently processing your responses to create a personalized path just for you. This is where the magic begins—we’re using your input to tailor the experience, offering you the guidance and motivation you need to achieve a balanced, fulfilling life.\n",
	"Hang tight while we set things up! In just a moment, you’ll dive into the areas that matter most to you, and together, we’ll start making progress toward your goals. Your journey to a better life starts now!",
];

```

Reply:
I placed a gap between TypingTextView components with adding gap to their parent View